### DISCUSSION TITLE
Tailwind styling not loading in MDX files.

### DISCUSSION THREAD

**Discussion by ahmetskilinc** [tailwindlabs/tailwindcss#9117] [Help] [answered]
Tailwind styling is not working in mdx files as expected, when wrapped in a class outside of the mdx file the styles load properly, but when in the mdx file they don't.

`tailwind.config.js`:
```javascript
module.exports = {
  content: ["./pages/**/*.{js,jsx}", "./components/**/*.{js,jsx}"],
  theme: {},
  plugins: [require("@tailwindcss/typography")],
};
```

`PostLayout.jsx`:
```jsx
const PostLayout= ({ children }) => {
  return (
    <article className="max-w-none prose prose-stone dark:prose-invert">
      {children}
    </article>
  );
};

export default PostLayout;
```

`why-i-use-vs-code.mdx`:
```mdx
---
title: Why Do I Use VS Code
---

import PostLayout from "../../layout/PostLayout";

<PostLayout>

# Why do I use VS Code

## What is VS Code

VS Code is a powerful text editor

</PostLayout>
```

**Comment by wongjn** [comment, marked as answer]
Could we take a look at what the resultant rendered HTML looks like when the styling does not apply? Also would double-check that `PostLayout.jsx` is covered by the Tailwind config `content` globs.

**Reply by ahmetskilinc** [reply]
WOW. I missed the layout folder from the content globs! Thank you for your comment and the reminder!

EDIT: Added `./layout/**/*.{jsx}` to `content` globs array.
